CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 🛑 Breaking changes 🛑
+- confighttp supports multiple compression methods including `gzip`, `zlib`, `deflate`, `snappy` and `zstd`.  (#4441)
```

Comment:
I don't think this is a breaking change, is it? I believe this should go into "Enhancements". Also, a suggestion for this entry would be:

```suggestion
- `confighttp`: add client-side compression support.  (#4441)
```

Reply:
fixed. I thought removing compression from otlphttpexporter will be a breaking change.